src/common/components/beta-notification/trigger.js

```diff
@@ -0,0 +1,48 @@
+import { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import localforage from 'localforage';
+
+import { BETA_NOTIFICATION_TOGGLE } from '../../reducers/beta-notification';
+import { BETA_NOTIFICATION_DISMISSED_KEY } from './index';
+
+const BETA_NOTIFICATION_DELAY = 2 * 6 * 1000; // 2 minutes
+
+export class BetaNotificationTriggerView extends Component {
+  notificationTimeout = null;
+
+  async componentDidMount() {
+    const notificationDismissed = await localforage.getItem(BETA_NOTIFICATION_DISMISSED_KEY);
+
+    if (this.props.authenticated && !notificationDismissed) {
+      this.notificationTimeout = setTimeout(this.props.showNotification, BETA_NOTIFICATION_DELAY);
```

Comment:
This dismisses the notification after 2 minutes if the user hasn't? 
2 minutes is a long time, am I correct is saying a user could potentially be further down the page and then the banner dismisses, in flow, the UI reflows and the user is wondering why?
Even if it didn't do that, what's the gain from timeout of this banner? It seems like work for no gain.